webpack.common.js

```diff
@@ -112,7 +112,13 @@ module.exports = {
         'adhocracy4/adhocracy4/categories/assets/select_dropdown_init.js'
       ],
       dependOn: 'adhocracy4'
-    }
+    },
+    budgeting_disable_contact: {
+      import: [
+        './meinberlin/apps/budgeting/assets/disable_contact.js'
+      ],
+      dependOn: 'adhocracy4'
```

Comment:
Do I need this here?

Reply:
This tells webpack that adhocracy4.js will always be present when this endpoint is used, allowing to avoid duplicates in imports.

Locking at `disable_contact.js` there appears to be only a implicit import: jquery (`$(document).ready`). It's likely a good idea to avoid jquery in all new code so if that's done - then this `dependOn` is not necessary.